### PR TITLE
Feature/pokedex display pokemon forms

### LINKED ISSUE #25: Form display
When a Pokemon is selected, add a new row to the Pokedex, below evolutions and above learnsets.

The row must display sprites of the base form and every alternative form. The sprites must be meaningfully smaller than that of the sprite in the heading, around the same as the font size in which the form text is in.

There's no need to make this display fancy, it can simply be a list of:
`[sprite <pokemon name>], [sprite <pokemon name>], [sprite <pokemon name>]`

### PR DESCRIPTION
**Pull request by Glup3**
# Motivation
We would like to display all alternative forms of a Pokemon. I also added a test to check for all missing Pokemon forms. Enhanced GitHub Action to display jest report.

# Solution

https://user-images.githubusercontent.com/25965213/233857164-c4af215c-2d55-497a-ad86-093941abf9cb.mp4

Closes #25

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "@mui/icons-material": "^5.11.0",
     "@mui/material": "^5.11.0",
     "clsx": "^1.2.1",
+    "fast-xml-parser": "^4.2.2",
```
Comment (Denmark14) [inline package.json:27]: Is there a reason we're using xml over json?

File: scripts/junitReportParser.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const { XMLParser } = require('fast-xml-parser');
+const fs = require('fs');
+const path = require('path');
+
+const parser = new XMLParser({ ignoreAttributes: false });
+const junitReportFilePath = path.join(__dirname, '../reports/jest-report.xml');
```
Comment (Denmark14) [inline scripts/junitReportParser.js:6]: If you want the parent filepath, try [`path.basename`](https://nodejs.org/api/path.html#path_path_basename_path_ext)

**Review by Denmark14 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Glup3) [reply package.json:27]: the generated jest-junit report is xml - just included the parser for the mini script

File: scripts/junitReportParser.js (hunk 2 above)
Comment (Glup3) [reply scripts/junitReportParser.js:6]: how does `path.basename` give me the parent filepath? the docs I read say that i need to give it whole path and it returns the filename. but i want to create the path.

**Pull request merged**